### ISSUE TITLE
Feature Request: Google Places API

### ISSUE THREAD

**Issue by nskeip** [brendanhay/gogol#81]
Would be nice to have an ability to use Google Places API with this library.
https://developers.google.com/places/

**Comment by korayal** [comment]
I don't think this is possible, since it's not available in [Google's GO API packages](https://github.com/googleapis/google-api-go-client) which this package uses to auto-generate. It may be labelled as something other than `places` though.

**Comment by tonicebrian** [comment]
It seems that [it is now available](https://github.com/googleapis/google-api-go-client/blob/main/places/v1/places-api.json). Marking this as an enhancement.
